src/app/component/dashboard/customers/customers.component.ts

```diff
@@ -299,6 +310,23 @@ export class CustomersComponent implements OnInit {
 
   previewUser(selectedUser) {
     console.log(selectedUser);
+    if (selectedUser.img && selectedUser.img.data.length !== 0) {
+      const TYPED_ARRAY = new Uint8Array(selectedUser.img.data);
```

Comment:
It will be good if we use the same method for showing pictures to move this in helpService and make a global method.